Jenkinsfile

```diff
@@ -1,71 +1,55 @@
 import org.kohsuke.github.GitHub
 
-def getIsCMSDeploy(ref) {
-  // check to see if this ref has already been built
-  // if it has then we are doing a cms deployment, not a full pipeline
-  return sh (script: "aws s3 ls s3://vetsgov-website-builds-s3-upload/${ref}/",
-             returnStatus: true) == 0
-}
-
-def runDeploy(jobName, ref) {
-  build job: jobName, parameters: [
-    booleanParam(name: 'notify_slack', value: true),
-    stringParam(name: 'ref', value: ref),
-  ], wait: false
-}
-
-def isCMSDeploy = false
-
-def getEnvNames(isCMSDeploy) {
-  def vetsgovEnvNames = [
-    'development', 'staging', 'production'
-  ]
-  def vagovEnvNames = [
-    'preview', 'vagovdev', 'vagovstaging', 'vagovprod'
-  ]
+env.CONCURRENCY = 10
 
-  if (isCMSDeploy) {
-    // only return va.gov envs
-    return vagovEnvNames
-  }
-  return vetsgovEnvNames + vagovEnvNames
-}
+VETSGOV_BUILDTYPES = [
+  'development',
+  'staging',
+  'production'
+]
 
-def envNames = [
-  // Vets.gov envs
-  'development', 'staging', 'production',
-  // VA.gov envs
-  'preview', 'vagovdev', 'vagovstaging', 'vagovprod'
+VAGOV_BUILDTYPES = [
+  'preview',
```

Comment:
Don't need this!

Reply:
Actually have that captured here, https://github.com/department-of-veterans-affairs/vets.gov-team/issues/15080!